datasets/mlsum/mlsum.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import nlp
+
+
+_CITATION = """\
+@article{scialom2020mlsum,
+  title={MLSUM: The Multilingual Summarization Corpus},
+  author={Scialom, Thomas and Dray, Paul-Alexis and Lamprier, Sylvain and Piwowarski, Benjamin and Staiano, Jacopo},
+  journal={arXiv preprint arXiv:2004.14900},
+  year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+We present MLSUM, the first large-scale MultiLingual SUMmarization dataset. 
+Obtained from online newspapers, it contains 1.5M+ article/summary pairs in five different languages -- namely, French, German, Spanish, Russian, Turkish. 
+Together with English newspapers from the popular CNN/Daily mail dataset, the collected data form a large scale multilingual dataset which can enable new research directions for the text summarization community. 
+We report cross-lingual comparative analyses based on state-of-the-art systems. 
+These highlight existing biases which motivate the use of a multi-lingual dataset. 
+"""
+_URL = "https://gitlab.lip6.fr/scialom/mlsum_data/-/raw/master/MLSUM/"
+_LANG = ["de", "es", "fr", "ru", "tu"]
+
+
+class MlsumConfig(nlp.BuilderConfig):
+    def __init__(self, data_url, **kwargs):
+        """BuilderConfig for MLQA
+
+        Args:
+          data_url: `string`, url to the dataset
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(MlsumConfig, self).__init__(version=nlp.Version("1.0.0",), **kwargs)
+        self.data_url = data_url
+
+
+class Mlsum(nlp.GeneratorBasedBuilder):
+
+    # TODO(mlqa): Set up version.
+    VERSION = nlp.Version("1.0.0")
+    BUILDER_CONFIGS = (
+        [
+            MlsumConfig(
+                name=lang,
+                data_url=_URL,
```

Comment:
maybe you remove `data_url` given that it's the same for all config and it's equal to `_URL`